kernels/ZendEngine3/extended/fcall.c

```diff
@@ -278,7 +278,11 @@ static int zephir_is_callable_check_class(const char *name, int name_len, zend_f
 		} else {
 			fcc->called_scope = called_scope;
 			if (!fcc->object) {
-				fcc->object = Z_OBJ(EG(current_execute_data)->This);
+				#if PHP_VERSION_ID >= 70100
+					fcc->object = zend_get_this_object(EG(current_execute_data));
```

Comment:
Original `zend_function_call` in PHP 7.1 are fixing `EG(current_execute_data)->This` without `fcc->object` and this thing can be "null", I think `fcc->object` is needed only for callable